parquet/src/arrow/buffer/bit_util.rs

```diff
@@ -84,7 +84,8 @@ mod tests {
             .iter()
             .enumerate()
             .rev()
-            .filter_map(|(x, y)| y.then(|| x))
+            .filter(|&(_x, y)| *y)
+            .map(|(x, _y)| x)
```

Comment:
```suggestion
            .filter_map(|(x, y)| y.then_some(x))
```
Perhaps, but perhaps it will complain about that as well

Reply:
Sorry about that! I guess I missed ticking the box that would let you modify my PR. I committed your suggestion!